.github/workflows/change-install-version.yaml

```diff
@@ -0,0 +1,31 @@
+name: Change the install version to match the latest tagged version
+on:
+  release:
+    types: [published]
+
+jobs:
+  # Update install version
+  update-install-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get version
+      run: |
+        tarref=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+        # Strip "v" prefix from tagref
+        if [[ "${{ github.ref }}" == "refs/tags/"* ]]; then
+          echo "VERSION=$(echo $tarref | sed -e 's/^v//')" >> $GITHUB_ENV
+        elif [[ "${{ github.ref }}" == *"master" ]]; then
+          echo "VERSION=latest" >> $GITHUB_ENV
+        fi
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: pip3 install -r ./.github/workflows/requirements.txt
+    - name: Edit install version
+      run: python3 ./.github/workflows/change-install-version.py ${{ env.VERSION }}
```

Comment:
I'm not understanding how this updates `mkdocs/docs/getting-started/install.md`. I see the py script updates only `mkdocs.yml`.

Reply:
I've replaced the install version with a variable. Now we can dynamically update the install version by modifying `mkdocs.yml`.